types/carbon-components-react/lib/components/FileUploader/FileUploader.d.ts

```diff
@@ -18,7 +18,8 @@ export type FileUploaderSize = "default" | "field" | "small";
 
 interface FileUploaderButtonInheritedProps extends
     Omit<ReactLabelAttr, "onChange">,
-    SharedProps
+    SharedProps,
+    ReactInputAttr
```

Comment:
This won't work with inheritance of `ReactLabelAttr` above. I'm guessing this was changed in some minor version? If that's the case, then it should also be considered that the version in the [index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/carbon-components-react/index.d.ts#L1) file should be updated. DT recommends that the package major and minor version match the version of the actual lib the types are for.